raster3d/r3.in.lidar/main.c

```diff
@@ -136,6 +136,30 @@ void binning_add_point(struct PointBinning3D *binning, int row, int col,
     }
 }
 
+static void cell_head_init(struct Cell_head *region)
+{
+    region->format = -1;
+    region->compressed = 0;
```

Comment:
Cell_head is defined here: https://github.com/OSGeo/grass/blob/09c629dee487247df4ac768294dd9bae5eb42b99/include/grass/gis.h#L436-L496

All of the other values set here look fine, the right type.

I'm just not sure myself about the first two. They have a meaning defined in the docs of the structure Cell_head.

Reply:
At least for compression, I went ahead with the one which says uncompressed by default.

In case of format, since we are dealing with '2D/3D raster maps', I felt assigning value of -1 (which the definition says is for FP raster maps which I am hoping is different from 2D/3D raster maps' is the right choice.

Reply:
The Cell_head structure is for 2D rasters, but it is reused for extends or more generally regions...or something like that. Here it is used for region (extent, possibly resolution, too), so format is not really important. Given that `memset(&region, 0, sizeof(struct Cell_head));` would probably suffice.

Reply:
> The Cell_head structure is for 2D rasters, but it is reused for extends or more generally regions...or something like that. Here it is used for region (extent, possibly resolution, too), so format is not really important. Given that `memset(&region, 0, sizeof(struct Cell_head));` would probably suffice.

Or using [G_zero()](https://grass.osgeo.org/programming8/defs_2gis_8h.html#a1a50fcbeca1a29ea3a0f0dc7bfe257a4) like:

https://github.com/OSGeo/grass/blob/09c629dee487247df4ac768294dd9bae5eb42b99/lib/raster/color_init.c#L25-L28

Reply:
I agree with this suggestion. I have used `G_zero()` to initialize the structure in multiple places. Thanks.

Reply:
I agree with your suggestion. I have modified the code accordingly! Thanks.